lib/service/businessLogic.js

```diff
@@ -55,7 +55,7 @@ function process(task, modules, callback) {
   }
 
   const logicHandler = resolve(task.taskName);
-  return logicHandler(task.request, businessLogicCompletionHandler);
+  return logicHandler(task.request, businessLogicCompletionHandler, modules);
```

Comment:
Need a test which ensures that the modules are actually available inside each `logicHandler`